request.go

```diff
@@ -716,14 +758,37 @@ func (req *SelectRequest) Key(key interface{}) *SelectRequest {
 	return req
 }
 
+// FetchPos determines whether to fetch positions of tuples. A position
```

Comment:
Isn't it more like "a position of the last tuple"?

Reply:
Thank you, updated.